Source/Urho3D/Core/Object.cpp

```diff
@@ -72,7 +87,8 @@ bool TypeInfo::IsTypeOf(const TypeInfo* typeInfo) const
 }
 
 Object::Object(Context* context) :
-    context_(context)
+    context_(context),
+    blockEvents_(false)

```

Comment:
I'm unsure that you really need this hack. Event suppression is usually better to make locally, shuting down undesired events one-by-one. Please, re-think this design decision. However, I'm not strictly against it.

Reply:
I implemented it for a game state switching. At times this is very convenient, at other times unsubscribing/resubscribing events is more suitable. Just one more tool giving more freedom to developer.